shard.yml

```diff
@@ -70,6 +70,10 @@ dependencies:
     github: amberframework/teeplate
     version: ~> 0.5.0
 
+  inflector:
```

Comment:
We already have this shard some lines below :sweat_smile:

Reply:
missed this ). Will fix later